scripts/ci-build-azure-ccm.sh

```diff
@@ -43,15 +45,25 @@ setup() {
     export IMAGE_REGISTRY=${REGISTRY}
     pushd "${AZURE_CLOUD_PROVIDER_ROOT}" && IMAGE_TAG=$(git rev-parse --short=7 HEAD) && export IMAGE_TAG && popd
     echo "Image Tag is ${IMAGE_TAG}"
+
+    if [[ -n "${WINDOWS_SERVER_VERSION:-}" ]]; then
+        if [[ "${WINDOWS_SERVER_VERSION}" == "windows-2019" ]]; then
+            export WINDOWS_IMAGE_VERSION="1809"
+        elif [[ "${WINDOWS_SERVER_VERSION}" == "windows-2022" ]]; then
+            export WINDOWS_IMAGE_VERSION="ltsc2022"
+        else
+            echo "Not support windows version: ${WINDOWS_SERVER_VERSION}"
```

Comment:
```suggestion
            echo "Windows version not supported: ${WINDOWS_SERVER_VERSION}"
```

Reply:
> 

Thanks for the review. Code updated. PTAL.